src/kit/LogViewer/LogViewerSelect.tsx

```diff
@@ -143,6 +155,11 @@ const LogViewerSelect: React.FC<Props> = ({
       {showSearch && (
         <Input placeholder="Search Logs..." value={filters.searchText} onChange={handleSearch} />
       )}
+      {onClickSearch && (
+        <Button onClick={onClickSearch}>
+          <Icon name={searchOn ? 'close' : 'search'} title="search" />
+        </Button>
+      )}
```

Comment:
nit: it's a little odd to have this _and_ `showSearch` _and_ also `showSearch` being false doesn't affect this showing up at all. It might be better to instead have a `preExtraOptions` prop where we just dump whatever react children we want here before the search input.

Reply:
I agree the current props setup is not ideal, but if we move the current search input out of the `LogViewerSelect` component, we would need to move the throttle functions that control the filters out too, while currently the filters are entirely controlled inside the component

Reply:
Added some comments for explanation.

Reply:
let's keep it for now, but i think:
1) the log viewer settings config living in hew where it's passed to `useSettings` and
2) the log viewer settings now containing some filters that are no longer fully controlled within the component
is a sign that we should probably reduce the surface area of the component.